Makefile

```diff
@@ -2,11 +2,11 @@ NODE_BIN=./node_modules/.bin
 
 help:
 	@echo '                                                                                     		'
-	@echo 'Makefile for the xqueue-watcher                                                   		'
+	@echo 'Makefile for the xqueue-watcher                                                   			'
```

Comment:
Please fix the extra spacing.  Perhaps some tabs got converted to spaces or vise-versa.